### PR TITLE
Added steps for IPv6 based deployment in inventory file

### PR DESCRIPTION
**Pull request by gobindadas**

### CODE REVIEW

File: playbooks/hc-ansible-deployment/gluster_inventory.yml [hunk 1]
```diff
@@ -305,6 +305,9 @@ hc_nodes:
 
   # Common configurations
   vars:
+    # In case of IPv6 based deployment "gluster_features_enable_ipv6" needs to be enabled, like:
+    # gluster_features_enable_ipv6: true
```
Comment (pkesavap) [inline playbooks/hc-ansible-deployment/gluster_inventory.yml]: Will the user uncomment this to enable ipv6?

File: playbooks/hc-ansible-deployment/gluster_inventory.yml [hunk 2]
```diff
@@ -402,3 +405,9 @@ gluster:
   storage_domains:
     - {"name":"data","host":"host1-frontend-network-FQDN","address":"host1-backend-network-FQDN","path":"/data","function":"data","mount_options":"backup-volfile-servers=host2-backend-network-FQDN:host3-backend-network-FQDN"}
     - {"name":"vmstore","host":"host1-frontend-network-FQDN","address":"host1-backend-network-FQDN","path":"/vmstore","function":"data","mount_options":"backup-volfile-servers=host2-backend-network-FQDN:host3-backend-network-FQDN"}
+
+# In case of IPv6 based deployment there is additional mount option required i.e. xlator-option="transport.address-family=inet6"
+# Ex:
+#storage_domains:
```
Comment (pkesavap) [inline playbooks/hc-ansible-deployment/gluster_inventory.yml]: does user have to uncomment this as well to enable the appropriate storage domain to be set?

File: playbooks/hc-ansible-deployment/gluster_inventory.yml (hunk 1 above)
Comment (gobindadas) [reply playbooks/hc-ansible-deployment/gluster_inventory.yml]: Corrected with next commit

File: playbooks/hc-ansible-deployment/gluster_inventory.yml (hunk 2 above)
Comment (gobindadas) [reply playbooks/hc-ansible-deployment/gluster_inventory.yml]: Corrected with next commit

**Pull request merged**